### DISCUSSION TITLE
Erro nos links de redirecionamento

### DISCUSSION THREAD

**Discussion by ANDERSONRN** [bluecaret/carettab#376] [Help and Support]
Gostaria de relatar um pequeno problema que afeta o visual da extensão. Onde ficam os links de redirecionamento, os ícones não estão mais aparecendo.

![print](https://github.com/bluecaret/carettab/assets/43056533/d1f8c833-32dc-472d-a06c-e21edb58c2c3)

**Comment by bluecaret** [comment]
The browser can lose the favicons it stores sometimes, especially if you clear your temporary files. You should just need to open that website again and the icon should show up soon.
